addons/mass_mailing/static/src/js/wysiwyg.js

```diff
@@ -20,6 +20,29 @@ const MassMailingWysiwyg = Wysiwyg.extend({
         return res;
     },
 
+    /**
+     * @override
+     */
+     getValue: function () {
+        let editable = this._super(...arguments);
```

Comment:
@shsa-odoo Did you review this ? I'm asking because:
- this is a `let` that is not reassigned so it should be a `const`
- `editable` is not a good name for this variable because the editable is an actual concept in the editor and wysiwyg and it's not supposed to be a string. It is correct to say that the string here _does_ in fact represent the editable but it is not the _actual_ editable, if you see what I mean, so I think naming the variable `editable` is not a good idea as people might easily get confused. Considering we are retrieving the return of the super call to the `getValue` method, I'd just call it `value`.

Maybe you didn't review the new version because we were making code changes while keeping the task in RD Review stage ? Would you prefer that developers take the task back to RD Dev ? Let me know what you think.

Reply:
Yes i didn't got the chance to review it before he was using that editable when i reviewed it, and also i agree, this code should be simply like this, as it we dont actually need to declare one variable named as document, when all we are coding for is an editable Zone, so IMO code should be something like this:

`getValue: function () {
    const initialValue = this._super(...arguments);
    const editableZone = (new DOMParser().parseFromString(initialValue, "text/html")).querySelector('.o_editable');
    return editableZone ? editableZone.innerHTML : initialValue;
}`
@dmo-odoo i thank you for your valuable suggestions and insights i will keep them in my mind whenever i will review the code. Its really good to learn.

Reply:
Based on your version, I suggest the following:
```js
getValue: function () {
    const value = this._super(...arguments);
    // Prevent users content outside of the dropzone.
    const dropzone = (new DOMParser().parseFromString(initialValue, "text/html")).querySelector('.o_editable');
    return dropzone ? dropzone.innerHTML : value;
}
```

We often need to make changes at RD Review stage so you reviewed the code once already, then I review, then we ask for changes sometimes, then ideally you should review again. How do you want to organize the process with your team to do that ?

Reply:
I believe if the changes is very minimal like naming conventions, which ideally will not have any functional anomaly, we can go with your suggested changes so that issues gets resolved ASAP.
Yes the best practice is we should take back the task to dev and again let it go through functional testing whether the changed code now has affected any functionality somewhere else, before we merge.
isn't it right?

Reply:
I agree. Who should make the decision to move the task back to RD Dev when the changes are significant ? You ? Me ? The dev who changes the code ?

Reply:
I think we should instruct the developers responsible in team that when the suggested changes are significant they must change the stage to dev when changes are requested from the reviewer.